Classes/CategorizeImages.cs

```diff
@@ -12,11 +12,11 @@
 namespace CategorizingImages{
     public static class ImageCategorizer{
 
-        public static List<CategorizedImage> Categorizer(List<Image> input){
-            MLContext myContext=new MLContext();
-            DataViewSchema modelSchema;
-
-            ITransformer trainedModel = myContext.Model.Load("model.zip", out modelSchema);
+        public static List<CategorizedImage> Categorizer(List<Image> input,ITransformer trainedModel, MLContext mycontext){
+           // MLContext myContext=new MLContext();
+            //DataViewSchema modelSchema;
+            //string modelName=CustomBuilder.GetModelNames();
```

Comment:
Hat du das getestet? Weil modelName selbst kein vollständiger Pfad ist

Reply:
Nein, in dem Branch ist dein Teil noch nicht drin...

Welche Variable aus Tools ist jetzt die richtige für die gespeicherten Modelle?

Reply:
ModelDir ist es nicht, da liegt nur das InceptionModel, aber das kann man erstmal nicht nutzen, ohne das zu trainieren

Reply:
Okay, ich merge erstmal, dann schauen wir